internal/output/output.go

```diff
@@ -346,27 +344,22 @@ func (r *Root) summaryMessage(showSkipped bool) string {
 			msg += fmt.Sprintf("\n∙ %d were estimated", *r.Summary.TotalSupportedResources)
 		}
 
+		allUsageBased := *r.Summary.TotalUsageBasedResources == *r.Summary.TotalSupportedResources
+
 		if r.Summary.TotalUsageBasedResources != nil && *r.Summary.TotalUsageBasedResources > 0 {
-			usageLink := ui.SecondaryLinkString("https://infracost.io/usage-file")
-			if *r.Summary.TotalUsageBasedResources == 1 {
-				msg += fmt.Sprintf(", 1 includes usage-based costs, see %s", usageLink)
-			} else {
-				msg += fmt.Sprintf(", %d include usage-based costs, see %s", *r.Summary.TotalUsageBasedResources, usageLink)
+			usageBasedCount := "from which 1 includes"
```

Comment:
"from which" or "of which"? I think "of which" is correct because you'd say "two of them".

Reply:
English is hard :) In this case we can put the number first, right? `1 of which includes`/`20 of which include`.